game/scripts/npc/items/custom/item_trumps_fists.txt

```diff
@@ -83,12 +83,12 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "heal_prevent_percent"                            "75"
+        "heal_prevent_percent"                            "60 70"
```

Comment:
I had Judecca on Zeus the other day. I wasn't doing any damage to your Batrider. Nerf it more and you wouldn't need any magic resistance to sustain the damage.